actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -87,6 +103,12 @@ def _compute_redirect_to_location(request, options) #:nodoc:
       when String
         request.protocol + request.host_with_port + options
       when :back
+        ActiveSupport::Deprecation.warn(<<-MESSAGE.squish)
+          `redirect_to :back` is deprecated and will be removed from Rails 5.1.
+          Please use `redirect_back(fallback: fallback_location)` where
+          `fallback_location` represents the redirect to use if the request has
```

Comment:
Yeah, I'll use location. It's not always a URL but rather something that will eventually be a URL. I think location is a better indicator of that.